mantle/cmd/kola/resources/iscsi_butane_setup.yaml

```diff
@@ -135,11 +138,14 @@ systemd:
         After=targetd.service
         ConditionFirstBoot=true
         OnFailure=emergency.target
+        OnFailureJobMode=isolate
         [Service]
         Type=oneshot
         RemainAfterExit=yes
         ExecStart=/usr/local/bin/targetcli_script
         Restart=on-failure
+        StandardOutput=journal
+        StandardError=journal
```

Comment:
I'm confused why we need this at all. Shouldn't it be captured in the journal anyway?

Reply:
in my previous testing it always have been captured in the journal yes

Reply:
I think maybe I was trying to iterate and figure out why I wasn't getting complete log output and thought this helped, but it was really https://github.com/coreos/coreos-assembler/pull/3789/commits/7fc1dd3b2943bd1748b418fb25e2947ec9f377ea that was the key to fixing that problem.

I can drop the `StandardOutput` and `StandardError` updates in the next upload.